CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - [#5405](https://github.com/apache/trafficcontrol/issues/5405) - Prevent Tenant update from choosing child as new parent
 - [#5384](https://github.com/apache/trafficcontrol/issues/5384) - New grids will now properly remember the current page number.
 - [#5548](https://github.com/apache/trafficcontrol/issues/5548) - Don't return a `403 Forbidden` when the user tries to get servers of a non-existent DS using `GET /servers?dsId={{nonexistent DS ID}}`
+- [#5724](https://github.com/apache/trafficcontrol/issues/5724) - Generate XMPPID on update if the server had none
```

Comment:
Since this PR no longer generates UUIDs for XMPPID, we should probably say `Set XMPPID to hostname if the server had none, don't error on server update when XMPPID is empty`.

Also, would the title would make more sense now as "Set server XMPPID to hostname if empty"?

Reply:
Updated PR title and updated changelog entry in 04f137b767.